src/Componants/TouristInfoCards.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+const TouristInfoCards = () => {
+  return (
+    <div className="card">
+      <div className="card-body">
+        <img
+          src="https://images.amcnetworks.com/bbcamerica.com/wp-content/uploads/2014/02/Glasgow-3.jpg"
+          className="card-img-top"
+        />
+        <a
+          href="https://peoplemakeglasgow.com"
+          target="_blank"
+          className="btn btn-primary"
+        >
+          Glasgow
+        </a>
+      </div>
```

Comment:
Don't you think this card could be a component? If we get some ideas from other parts of your PR, we could have an array of cardInfo, map in here and generate this code instead of repeat it. 

```js
const cardInfos = [{
  image: 'https://images.amcnetworks.com/bbcamerica.com/wp-content/uploads/2014/02/Glasgow-3.jpg',
  URL: 'https://peoplemakeglasgow.com',
  cityName: 'Glasgow'
}, {
  image: 'https://media.timeout.com/images/105400776/630/472/image.jpg,
  URL: 'https://visitmanchester.com,
  cityName: 'Manchester'
}]
```

What do you think?